api/src/pcapi/routes/backoffice_v3/accounts.py

```diff
@@ -939,6 +945,24 @@ def get_public_account_link(user_id: int, **kwargs: typing.Any) -> str:
     return url_for("backoffice_v3_web.public_accounts.get_public_account", user_id=user_id, **kwargs)
 
 
+def get_fraud_check_concerned_eligibility(
+    user: users_models.User, fraud_check: fraud_models.BeneficiaryFraudCheck
+) -> users_models.EligibilityType | None:
+    if fraud_check.eligibilityType:
+        return fraud_check.eligibilityType
+
+    if not user.dateOfBirth:
+        return None
+
+    age_at_fraud_check_date = users_utils.get_age_at_date(user.dateOfBirth, fraud_check.dateCreated)
+    concerned_eligibility = (
+        users_models.EligibilityType.UNDERAGE if age_at_fraud_check_date < 18 else users_models.EligibilityType.AGE18
+    )
+    fraud_check.eligibilityType = concerned_eligibility
+
+    return concerned_eligibility
+
+
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Plusieurs choses

- Ça a du sens de mettre ça dans le modèle du fraud check, plutôt que dans le BO
- Ça doit utiliser `applicable_eligibilities` et `user.eligibility`sinon on duplique de la logique métier cruciale 
- Je pense que le naming se challenge

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏
```python
class BeneficiaryFraudCheck(PcObject, Base, Model):
    [...]
    @property
    def get_eligibility_at_creation(self) -> users_models.EligibilityType | None:
        if self.eligibilityType:
            return self.eligibilityType

        user_eligibility_at_fraud_check_creation = self.user.get_eligibility_at_date(
            self.user.birth_date, self.dateCreated
        )
        if user_eligibility_at_fraud_check_creation in self.applicable_eligibilities:
            return user_eligibility_at_fraud_check_creation

        return None
```
Ensuite tu peux utiliser `fraud_check.get_eligibility_at_creation()`
Je suis paas 100% convaincu par le nom `get_eligibility_at_creation`

Reply:
Ah j'avoue je n'y avais pas pensé ! Merci :)

Reply:
Je vais plutôt le nommer en `get_target_eligibility_at_creation` ou un truc du genre si ca te va. 
Et changer un peu la logique pour que cela me renvoie forcément une éligibilité. C'est ce qui est un peu tordu parce que dans le cas d'un tunnel `underage` ET `age18`, on doit pouvoir positionner la `fraud check` sans `eligibilityType` dans un des deux tableaux.

Reply:
En fait j'hésite même à l'associer au modèle sachant que c'est vraiment juste pour l'affichage tu en penses quoi ?